x-pack/plugins/security_solution/server/usage/detections/rules/types.ts

```diff
@@ -45,3 +46,89 @@ export interface RuleMetric {
   has_legacy_notification: boolean;
   has_notification: boolean;
 }
+
+export interface SingleEventLogStatusMetric2 {
```

Comment:
Frank what is this, `SingleEventLogStatusMetric2`? Is that left over? I would remove this.